### ISSUE TITLE
Flex to ANTLR converter

### ISSUE THREAD

**Issue by wasabii** [kaby76/AntlrExamples#1]
Do you happen to have a Flex to ANTLR lexer converter, like you do for Bison?

**Comment by kaby76** [comment]
No, I don't. Flex conversion is more complicated due to states and action code. I decided to postpone this for some later time. I'll revisit Flex after polishing other parts of Antlrvsix.

**Comment by Zireael07** [comment]
Did this ever materialize?
